src/ring_theory/norm.lean

```diff
@@ -326,6 +334,41 @@ begin
   { apply_instance }
 end
 
+variables {F} (L)
+
+lemma norm_norm [algebra L F] [is_scalar_tower K L F] [is_separable K F]
```

Comment:
I don't know the math here; is this obvously only true under `is_separable`? Or is that just what this proof approach needed?

Reply:
It seems that the proofs under non-separable assumptions are a lot harder: https://math.stackexchange.com/questions/1569032/to-prove-that-the-norm-of-a-tower-of-field-extensions-is-the-composition-of-norm

Reply:
What you need first is a generalisation of [algebra.norm_eq_prod_embeddings](https://leanprover-community.github.io/mathlib_docs/ring_theory/norm.html#algebra.norm_eq_prod_embeddings) to the non separable case. That's already non trivial...

Reply:
Here's a version for rings: https://math.stackexchange.com/q/2972232/1896

If you want to stick with the field proof, can you add a TODO to generalize to rings following that proof?

Reply:
All the proofs I could find need either results on inseparable extensions that are missing in mathlib or delicate matrix computations. I'll leave the proof as it is and add the TODO note you suggested.